### DISCUSSION TITLE
Determine the tolerance for combustion simulations

### DISCUSSION THREAD

**Discussion by jiweiqi** [DENG-MIT/Arrhenius.jl#53] [Q&A]
References
* https://github.com/ReactionMechanismGenerator/ReactionMechanismSimulator.jl/issues/87

**Comment by jiweiqi** [comment]
Experience from Chemkin
![image](https://user-images.githubusercontent.com/8445647/111895890-13a80e00-89ec-11eb-8d82-8d8ebf5382cc.png)
New setting as R2021
![image](https://user-images.githubusercontent.com/8445647/111895902-2884a180-89ec-11eb-81ff-3a5452dba6ba.png)
Old setting
![image](https://user-images.githubusercontent.com/8445647/111895948-6f729700-89ec-11eb-901a-b49c95705a2e.png)

**Reply by TJP-Karpowski** [reply]
I agree with that, I would use similar settings in my flame calculations.
